Formula/libcdio-paranoia.rb

```diff
@@ -0,0 +1,22 @@
+class LibcdioParanoia < Formula
+  desc "Audio CD ripper based on libcdio"
+  homepage "https://www.gnu.org/software/libcdio/"
+  url "https://ftp.gnu.org/gnu/libcdio/libcdio-paranoia-10.2+0.94+2.tar.gz"
+  mirror "https://ftpmirror.gnu.org/libcdio/libcdio-paranoia-10.2+0.94+2.tar.gz"
+  sha256 "d60f82ece97eeb92407a9ee03f3499c8983206672c28ae5e4e22179063c81941"
+
+  depends_on "pkg-config" => :build
+  depends_on "libcdio"
+
+  def install
+    system "./configure", "--without-versioned-libs",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_match /cdparanoia/, shell_output("#{bin}/cd-paranoia -V 2>&1", 0).partition(" ").first
```

Comment:
We are going to need to test some operation. If nothing is possible, at least try to do something and check that we get the correct error (exit code and error message).

Reply:
This check is similar to the one from the cdparanoia formula. This library and binary is functionally the same as cdparanoia, but the main difference is that this is being actively maintain and it is supported in multiple platforms as it uses libcdio, which already exist in homebrew, to abstract all cd operations.

Reply:
Barrier is higher for tests on new formulas, than existing tests (which we're trying to improve over time, but we have lots of formulas).

Reply:
There is nothing noted in the documentation about this and there is not a lot of guidance on what are the requirements either.

Reply:
Try to perform some operation, and check that it fails. See for example formulas like `fabio`,  `gist`, `unravel`, etc.